frontend/src/styles/global.sass

```diff
@@ -6,9 +6,13 @@
 body
     font-family: "DM Sans"
     overflow-x: hidden
-    
-*:focus
+
+*:focus,
+.p-focus
     box-shadow: none !important
 
 html
     scroll-behavior: smooth
+
+strong
```

Comment:
You can use HTML tag for this

Reply:
Our reset.css adds a strong tag  "font: inherit" style and it won't bold. And then I return this property to the tag here.